CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 - Add support for the Address Ownership Proof Protocol (AOPP), i.e.: handle 'aopp:?...' URIs. See https://aopp.group/.
+- Add fee options based on priority, and the ability to set a custom gas price.
```

Comment:
I'd mention this is for etherium. the only hint is "gas" but some might not even know what gas is if they don't use ETH.

Reply:
Of course - how could I miss it...